test/js/bun/glob/__snapshots__/scan.test.ts.snap

```diff
@@ -1,580 +0,0 @@
-// Bun Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
you should revert all changes to unrelated test snapshots. i think windows has some bugs with messing with these.

Reply:
Reverted all snapshot changes.